### DISCUSSION TITLE
plexupdate fails to update plex.

### DISCUSSION THREAD

**Discussion by towerhand** [tteck/Proxmox#1424] [Help Needed] [answered]
### A clear and concise description of the issue.

 Hello tteck, I noticed Plex has been showing an update available for a few days, when it used to update automatically. I tried using the script: bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/plex.sh)" and it is unable to run plexupdate. Thank you for all your scripts, and maintaining them.

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

Directory to install into: /opt/plexupdate
'/opt/plexupdate' appears to contain a different git repository, cannot continue

[ERROR] in line 70: exit code 1: while executing command bash -c "$(wget -qO - https://raw.githubusercontent.com/mrworf/plexupdate/master/extras/installer.sh)"

root@plex:~# 
___________
Also tried installing it on a new folder and get this error:

Directory to install into: /opt/plexupdate_new
'/opt/plexupdate_new' doesn't exist, attempting to create... done
Installing plexupdate into '/opt/plexupdate_new'... done

Do you want to install the latest PlexPass releases? (requires PlexPass account) [Y/n] Y

Would you like to automatically install the latest release when it is downloaded? [Y/n] n

Writing configuration file '/etc/plexupdate.conf'... done

Would you like to set up automatic daily updates for Plex? [Y/n] y

Do you want to log the daily update runs to syslog so you can examine the output later? [Y/n] y

Should cron send you an email if an update is available/installed? [Y/n] n

Writing configuration file '/etc/plexupdate.cron.conf'... done

Installing daily cron job... done

Configuration complete. Would you like to run plexupdate with these settings now? [Y/n] y
Retrieving list of available distributions
Downloading release "plexmediaserver_1.32.2.7002-86cfcc10c_amd64.deb"
/tmp/plexmediaserver_1.32.2.7002-86cfcc10c 100%[=====================================================================================>]  72.07M  95.4MB/s    in 0.8s    
File downloaded

[ERROR] in line 70: exit code 10: while executing command bash -c "$(wget -qO - https://raw.githubusercontent.com/mrworf/plexupdate/master/extras/installer.sh)"

root@plex:~#

**Comment by tteck** [comment, marked as answer]
Info on how to use [plexupdate](https://github.com/mrworf/plexupdate/)

**Reply by towerhand** [reply]
It has been working for a long time but broke a few days ago. However, I added some lines to the conf file from the repo you shared, and it appears to be working again. Thanks.

**Comment by oOStroudyOo** [comment]
Not sure how you've got it set up, but I used the tteck script and it's never failed to update.

I do it via `apt update && apt upgrade`
